packages/cmrjs/tests/cmr-utils/test-mergeURLs.js

```diff
@@ -226,8 +226,8 @@ test('Does Complicated merging', (t) => {
     {
       URL: 's3://new/path/to/metadata.cmr.xml',
       URLDescription: 'from Updated',
-      Type: 'keeps Type: ORIGINAL',
-      MimeType: 'also from original: text/xml',
+      Type: 'GET DATA VIA DIRECT ACCESS',
```

Comment:
Is this correct, why did the cmr.xml file get a "GET DATA VIA DIRECT ACCESS" type on it to start with, and should it have `Type: 'ORIGINAL Type'` still?

Reply:
I'm not sure I fully understand, but I think I'll trust you because that function has gotten a bit complicated.